includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -528,7 +528,7 @@ public function javascript_params() {
 		$stripe_params['saveAppearanceNonce'] = wp_create_nonce( 'wc_stripe_save_appearance_nonce' );
 
 		// ECE feature flag
-		$stripe_params['isECEEnabled'] = WC_Stripe_Feature_Flags::is_stripe_ece_enabled();
+		$stripe_params['isECEEnabled'] = true;
```

Comment:
This is not used anymore. There is a variable with the same name as this, but it's the one defined in the comment above (the only use of `is_ece_enabled`):
https://github.com/woocommerce/woocommerce-gateway-stripe/blob/2c0ad609a52d2ca40b44562570340044e7720aec/client/entrypoints/express-checkout-settings/express-checkout-settings-section.js#L273-L284